kernel/src/snapshot.rs

```diff
@@ -472,6 +481,47 @@ mod tests {
         assert!(cp.is_none())
     }
 
+    fn valid_last_checkpoint() -> Vec<u8> {
+        r#"{"size":8,"size_in_bytes":21857,"version":1}"#.as_bytes().to_vec()
```

Comment:
you can shorten this to `b#"` instead to make a byte slice and then do `.to_vec()`

Reply:
hm don't think I can do `b#`?

Reply:
decided to leave as-is since doing `b` requirues escaping quotes